theano/sandbox/linalg/ops.py

```diff
@@ -1,66 +1,34 @@
 from __future__ import absolute_import, print_function, division
 import logging
-
-logger = logging.getLogger(__name__)
```

Comment:
The other theano sub-modules will have their own logger, so that is fine.
The logger is only used explicitly, for instance the calls to `logger.debug(...)` later in the same file.